test/NuGet.Clients.Tests/NuGet.VisualStudio.Test/IVsPathContextProviderTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Moq;
+using Xunit;
+
+namespace NuGet.VisualStudio.Test
+{
+    public class IVsPathContextProviderTests
+    {
+        // Verify interface method can be used across assembly boundaries
+        [Fact]
+        public void TryCreateContext_EmbeddableTest()
+        {
+            var provider = Mock.Of<IVsPathContextProvider>();
```

Comment:
I can't see any benefit in adding obvious comments. In my tests "Act" is a single line with a method call matching the test name. All "Arrange" code is placed above it, and "Assert" after it.